index.html

```diff
@@ -65,7 +65,131 @@ <h2 class="main-title main-section-subtitle">I'm a software<br> developer</h2>
+
+    <!--Work section-->
+    <section class="work-section-container">
+        <h1 class="work-section-title">My Recent<br>Work</h1>
+        <hr class="hr-line">
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+    </section>
+    <!--end of work section-->
+
+    <!--about me section-->
+    <section class="aboutme-container">
+        <img class="frame1" src="assets/images/Frame1.png" alt="frame 1">
+        <img class="frame2" src="assets/images/Frame2.png" alt="frame 2">
```

Comment:
- [ ] It looks like you made a typo writing the name of your image because right now you cannot see this image properly, instead, the screen is showing the `alt` text. I think it's just a problem with the capital F because your file is named `frame2` not `Frame2`.

Reply:
hi Lordkaito,
thank you for feedback, 
all note is modified, 
looking forward to another review 😎😃